sys/kern/subr_module.c

```diff
@@ -46,10 +46,14 @@
 vm_offset_t preload_addr_relocate = 0;
 caddr_t preload_metadata, preload_kmdp;
 
+const char *preload_modtype = MODTYPE;
```

Comment:
Why not `const char preload_modtype[] = MODTYPE'` ?

Reply:
Oh, that would be better yes